msgid ""
msgstr ""
"Project-Id-Version: Formulator trunk"
"POT-Creation-Date: Thu Jun 26 12:40:21 2008\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: Silva Translators <eric+i18n@infrae.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generated-By: zope/app/translation_files/extract.py\n"

#: /Validator.py:106
#: /Validator.py:496
#: /Validator.py:691
msgid "Input is required but no input given."
msgstr ""

#: /Validator.py:154
msgid "Too much input was given."
msgstr ""

#: /Validator.py:177
msgid "You did not enter an email address."
msgstr ""

#: /Validator.py:222
msgid "The entered value did not match the pattern."
msgstr ""

#: /Validator.py:277
msgid "You did not enter an integer."
msgstr ""

#: /Validator.py:278
msgid "The integer you entered was out of range."
msgstr ""

#: /Validator.py:308
msgid "You did not enter a floating point number."
msgstr ""

#: /Validator.py:365
msgid "You entered too many lines."
msgstr ""

#: /Validator.py:366
msgid "A line was too long."
msgstr ""

#: /Validator.py:367
msgid "You entered too many characters."
msgstr ""

#: /Validator.py:441
#: /Validator.py:497
msgid "You selected an item that was not in the list."
msgstr ""

#: /Validator.py:608
msgid "The specified link is broken."
msgstr ""

#: /Validator.py:692
msgid "You did not enter a valid date and time."
msgstr ""

#: /Validator.py:693
msgid "The date and time you entered were out of range."
msgstr ""

#: /Validator.py:83
msgid "The input failed the external validator."
msgstr ""

